{"schema_version":"1.6.0","id":"OESA-2024-1653","modified":"2024-05-24T11:32:48Z","published":"2024-05-24T11:32:48Z","related":["CVE-2024-3651"],"summary":"python-idna security update","details":"A library to support the Internationalised Domain Names in Applications (IDNA) protocol as specified in RFC 5891 http://tools.ietf.org/html/rfc5891. This version of the protocol is often referred to as “IDNA2008” and can produce different results from the earlier standard from 2003.\r\n\r\nSecurity Fix(es):\r\n\r\nA flaw was found in the python-idna library. A malicious argument was sent to the idna.encode() function can trigger an uncontrolled resource consumption, resulting in a denial of service.(CVE-2024-3651)","affected":[{"package":{"ecosystem":"openEuler-20.03-LTS-SP4","name":"python-idna","purl":"pkg:rpm/openEuler/python-idna&distro=openEuler-20.03-LTS-SP4"},"ranges":[{"type":"ECOSYSTEM","events":[{"introduced":"0","fixed":"2.10-2.oe2003sp4"}]}],"ecosystem_specific":{"noarch":["python3-idna-2.10-2.oe2003sp4.noarch.rpm","python2-idna-2.10-2.oe2003sp4.noarch.rpm"],"src":["python-idna-2.10-2.oe2003sp4.src.rpm"]}}],"references":[{"type":"ADVISORY","url":"https://www.openeuler.org/en/security/safety-bulletin/detail.html?id=openEuler-SA-2024-1653"},{"type":"ADVISORY","url":"https://nvd.nist.gov/vuln/detail/CVE-2024-3651"}],"database_specific":{"severity":"Medium"}}
